### DISCUSSION TITLE
How to break down the ingest url?

### DISCUSSION THREAD

**Discussion by Geo-GmZ** [datarhei/restreamer#413] [General]
Hi All. 

I'm trying to use the new version of Restreamer V2 with SRT and a 3rd party vendor. The 3rd party will be sending the stream to the Restreamer server, but it requires breaking down the stream URL as "Stream Key/ID," and "Passphrase" have their own fields.

Since these Ingest URLs are now auto-generated how can I tell with the URL below as an example?

 srt://**<PUBLICIP>**:6000?mode=caller&transtype=live&streamid=#!:m=publish,r=ingest/81da5f8c-ecd0-4d19-84ac-6096a5c19985

For RTMP using **"ingest/81da5f8c-ecd0-4d19-84ac-6096a5c19985"** as StreamKey seems to work but that doesn't seems to be the case with SRT.
 
Thanks in advance.

**Comment by jstabenow** [comment]
The SRT syntax is a little strange. See here:
https://haivision.github.io/srt-rfc/draft-sharabayko-srt.html#appendix-B.2

In your case, StreamKey would be `#!:m=publish,r=ingest/81da5f8c-ecd0-4d19-84ac-6096a5c19985`. 

We will check in a future release if we can implement an additional and comparable syntax to RTMP.
